fission/src/ui/panels/simulation/AutoTestPanel.tsx

```diff
@@ -159,7 +159,7 @@ function resetBodies(captures: BodyCapture[]) {
     JOLT.destroy(zero)
 }
 
-function End({ assembly, setStaging, captures }: EndProps) {
+const End: React.FC<EndProps> = ({ assembly, setStaging, captures }) => {
```

Comment:
I think biome automatically converted these from anonymous functions to regular ones but I much prefer it this way